pkgs/by-name/go/gotenberg/package.nix

```diff
@@ -0,0 +1,100 @@
+{
+  lib,
+  buildGoModule,
+  chromium,
+  fetchFromGitHub,
+  libreoffice,
+  makeWrapper,
+  pdftk,
+  qpdf,
+  unoconv,
+  mktemp,
+  makeFontsConf,
+  liberation_ttf_v2,
+  jre17_minimal,
+  exiftool,
+  python311,
+  nix-update-script,
+}:
+let
+  fontsConf = makeFontsConf { fontDirectories = [ liberation_ttf_v2 ]; };
+  jre' = jre17_minimal.override {
+    modules = [
+      "java.base"
+      "java.desktop"
+      "java.logging"
+      "java.sql"
+    ];
+  };
+  libreoffice' = "${libreoffice}/lib/libreoffice/program/soffice.bin";
+  unoconv' = unoconv.override { python3 = python311; };
+in
+buildGoModule rec {
+  pname = "gotenberg";
+  version = "8.8.0";
+
+  src = fetchFromGitHub {
+    owner = "gotenberg";
+    repo = "gotenberg";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-OHvtdUFZYm+I8DN8lAdlNAIRLWNRC/GbVEueRfyrwDA=";
+  };
+
+  vendorHash = "sha256-w9Q3hK8d5NwDYp8kydrWrlBlOmMyDqDixdv3z79yhK8=";
+
+  patchPhase = ''
+    find ./pkg -name '*_test.go' -exec sed -i -e 's#/tests#${src}#g' {} \;
+  '';
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/gotenberg/gotenberg/v8/cmd.Version=${version}"
+  ];
+
+  checkInputs = [
+    chromium
+    libreoffice
+    pdftk
+    qpdf
+    unoconv'
+    mktemp
+    jre'
+  ];
+
+  preCheck = ''
+    export CHROMIUM_BIN_PATH=${lib.getExe chromium}
+    export PDFTK_BIN_PATH=${lib.getExe pdftk}
+    export QPDF_BIN_PATH=${lib.getExe qpdf}
```

Comment:
This currently casues evaluation to [fail](https://gist.github.com/GrahamcOfBorg/a80c804b2a211250d938ae07e7b67f3d) due to `qpdf` not having a `meta.mainProgram` attribute. https://github.com/NixOS/nixpkgs/pull/326390 will fix this

Reply:
Will leave this till the other one is merged.

Reply:
Above PR merged, resolving.